commands/account/clean.ts

```diff
@@ -94,22 +106,50 @@ export async function handler(
         { border: { bodyLeft: '  ' } }
       )
     );
+
+    let promptMessage = i18n(
+      oneAccountFound ? `${i18nKey}.confirm.one` : `${i18nKey}.confirm.other`,
+      {
+        count: accountsToRemove.length,
+      }
+    );
+
+    const accountOverride = getCWDAccountOverride();
+    const overrideFilePath = getDefaultAccountOverrideFilePath();
+    if (overrideFilePath && accountOverride) {
+      if (
```

Comment:
Nit: These if statements can be merged.  Secondary nit:  The call to `.some` is a little long and might benefit from being stored in a variable